mri/optimizers/forward_backward.py

```diff
@@ -63,7 +63,7 @@ def fista(gradient_op, linear_op, prox_op, cost_op,
     metrics: dict
         the requested metrics values during the optimization.
     """
-    start = time.clock()
+    start = time.perf_counter()
```

Comment:
I am not necessarily against those test changes but maybe just reflect those in the PR title and description

Reply:
Will do it, this change had to be done as python3.8 has deprecated `time.clock()`